config/filter.d/bitwarden.conf

```diff
@@ -0,0 +1,9 @@
+# Fail2Ban filter for Bitwarden
+# Detecting failed login attempts
+# Logged in bwdata/logs/identity/Identity/log.txt
+
+[Init]
+datepattern = ^%%Y-%%m-%%d %%H:%%M:%%S
```

Comment:
I don't think it is valid pattern, since the log-lines seem to have timezones.
Also I don't see it is expected at all, since our default patterns (in 0.10) completely cover this dates.
So please remove it.

Reply:
Also note that verbose output of fail2ban-regex in normal case (considering correct TZ) would return following times for your test-lines from `tests/files/logs/bitwarden`:
```
Failregex: 2 total
|-  #) [# of hits] regular expression
|   1) [2] ^\s*\[WRN\]\s+Failed login attempt(, 2FA invalid)?\. <HOST>$
|      192.168.0.16  Mon Nov 25 18:04:49 2019
|      192.168.0.21  Mon Nov 25 21:39:58 2019
`-
```

Reply:
Thank you, let's then use the default date/time pattern.
I also updated the test-lines according to their TZ.